lib/Conversion/TorchToTcp/Utils.cpp

```diff
@@ -79,7 +85,7 @@ Value torch_to_tcp::broadcastInLeadingDimsToMatchShape(
 
 Value torch_to_tcp::broadcast0DOr1DToNDAndMatchShape(
     ConversionPatternRewriter &rewriter, Value input, Value target,
-    int64_t axisInOutput) {
+    int64_t axisInOutput, bool useInputAsResultType) {
```

Comment:
I'm not sure I understand why we need the `useInputAsResultType` bool. Instead, can we just create the target type in the callsite and send that in as a parameter?